clients/go/api.go

```diff
@@ -7,6 +7,7 @@ import (
 type ZBClient interface {
 	NewHealthCheckCommand() *commands.HealthCheckCommand
 	NewDeployWorkflowCommand() *commands.DeployCommand
+    NewCreateInstanceCommand() commands.CreateInstanceCommandStep1
```

Comment:
Can we auto format our go code? I saw multiple times different indention use inside the code base

Reply:
Strange, looks like Goland update forces new formatting defaults. I will gofmt entire project again. We should hook it up on git hook.